docs-gb/kubernetes/hpa-rps-autoscaling.md

```diff
@@ -13,6 +13,16 @@ and servers (single-model serving). This will require:
 * Configuring HPA manifests to scale Models and the corresponding Server replicas based on the
   custom metrics
 
+{% hint style="warning" %}
+The Core 2 HPA-based autoscaling has the following constraints/limitations:
+
+- HPA-scaling only works for single-model serving (1-1 correspondence between models and servers). Multi-model serving autoscaling is supported via the existing features described [here](autoscaling.md). Those continue to be improved targeting seamless autoscaling of a wider set of models and workloads.
```

Comment:
```suggestion
- HPA scaling is designed specifically for single-model serving, where there is a 1:1 correspondence between models and servers. Autoscaling for multi-model serving is supported by the existing [autoscaling feature](autoscaling.md). Ongoing improvements are focused on enhancing seamless autoscaling for a broader range of models and workloads.
```

Reply:
Will rephrase taking into account suggestions. Suggested rephrasing has parts that are incorrect or confusing from a technical perspective.